lib/ansible/executor/action_write_locks.py

```diff
@@ -36,8 +37,7 @@
 
     # These plugins are called directly by action plugins (not going through
     # a strategy).  We precreate them here as an optimization
-    mods = set(p['name'] for p in Facts.PKG_MGRS)
+    mods = set(C.PKG_MGR_MODULES)
```

Comment:
I think the PKG_MGR_MODULES data can live elsewhere, though constants.py seems reasonable to me (DEFAULT_SQUASH_ACTIONS lives there which seems similar).

The main requirement is getting it out of module_utils and facts.py to avoid the dependency from app into module_utils.facts.